src/platform/amd/renoir/include/platform/chip_registers.h

```diff
@@ -0,0 +1,571 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2021 AMD.All rights reserved.
+ *
+ * Author:      Basavaraj Hiregoudar <basavaraj.hiregoudar@amd.com>
+ *              Anup Kulkarni <anup.kulkarni@amd.com>
+ *              Bala Kishore <balakishore.pati@amd.com>
+ */
+#if !defined(_rn_REG_HEADER)
+#define _rn_REG_HEADER
+
+typedef	union acp_dma_cntl_0 {
+	struct {
+	unsigned int	dmachrst:1;
+	unsigned int	dmachrun:1;
+	unsigned int	dmachiocen:1;
+	unsigned int	: 29;
+	} bits;
+unsigned int	u32all;
+} acp_dma_cntl_0_t;
+
+typedef	union acp_dma_ch_sts {
+	struct {
+	unsigned int	dmachrunsts:8;
+	unsigned int	:24;
+	} bits;
+unsigned int	u32all;
+} acp_dma_ch_sts_t;
+
+typedef	union acp_external_intr_enb {
+	struct {
+	unsigned int	acpextintrenb:1;
+	unsigned int	:31;
+	} bits;
+unsigned int	u32all;
+} acp_external_intr_enb_t;
+
+typedef	union acp_dsp0_intr_cntl {
+	struct {
+	unsigned int	dmaiocmask:8;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	wov_dma_intr_mask:1;
+	unsigned int	:1;
+	unsigned int	:2;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	audio_buffer_int_mask:6;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	:1;
+	} bits;
+unsigned int	u32all;
+} acp_dsp0_intr_cntl_t;
+
+typedef	union acp_dsp0_intr_stat {
+	struct {
+	unsigned int	 dmaiocstat:8;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	wov_dma_stat:1;
+	unsigned int	:1;
+	unsigned int	:2;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	audio_buffer_int_stat:6;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	:1;
+	} bits;
+unsigned int	u32all;
+} acp_dsp0_intr_stat_t;
+
+typedef	union acp_dsp_sw_intr_cntl {
+	struct {
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	dsp0tohostintrmask:1;
+	unsigned int	:1;
+	unsigned int	:28;
+	} bits;
+unsigned int	u32all;
+} acp_dsp_sw_intr_cntl_t;
+
+typedef	union acp_dsp_sw_intr_stat {
+	struct {
+	unsigned int	hosttodsp0intr1stat:1;
```

Comment:
agree with @plbossart these are quite hard to read. best to use more ```_```

Reply:
Addressed the Review Comment. Done